### PR TITLE
FF128 HTTP Priority - RFC9218 Extensible Prioritization Scheme for HTTP

### PR DESCRIPTION
**Pull request by hamishwillee**
FF128 Adds support for [RFC 9218 Extensible Prioritization Scheme for HTTP](https://www.rfc-editor.org/rfc/rfc9218.html#name-the-priority-http-header-fi) in https://bugzilla.mozilla.org/show_bug.cgi?id=1865040.

This delivers a `Priority` header that can be sent in any HTTP version to provide a client hint about the relative priority that a resource should be sent by a server, which is added as a subfeature here. 
This is also supported in Chrome 124, which you can get from https://chromestatus.com/feature/5109106573049856 and the [release notes](https://developer.chrome.com/release-notes/124).
There is no indication of support in Safari or Webkit looking briefly through the release notes and doing a search, so I have put as false.

This is more than just the header as it also adds HTTP/2 and HTTP/3 frames to allow a hop-to-hop update of the priority expectations after the header has been sent. However we don't document these AFAIK so I don't have any way to indicate those are also supported as part of this (if I need to, let me know how).

Related docs work can be tracked in https://github.com/mdn/content/issues/33847

### CODE REVIEW

File: http/headers/Priority.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
+  "http": {
+    "headers": {
+      "Expect": {
```
Comment (Elchi3) [inline http/headers/Priority.json:4]:
```suggestion
      "Priority": {
```

File: http/headers/Priority.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+              "version_added": "124"
+            },
+            "chrome_android": "mirror",
+            "edge": "mirror",
+            "firefox": {
+              "version_added": "128"
+            },
+            "firefox_android": "mirror",
+            "ie": {
+              "version_added": null
```
Comment (Elchi3) [inline http/headers/Priority.json:19]:
```suggestion
              "version_added": false
```

**Review by Elchi3 — CHANGES_REQUESTED** [review]
Thank you!

File: http/headers/Priority.json (hunk 1 above)
Comment (hamishwillee) [reply http/headers/Priority.json:4]: Doh

**Comment by hamishwillee** [comment]
@Elchi3 Thank you. Sorry, that was sloppy. Accepted both changes.

**Comment by Elchi3** [comment]
I've filed https://github.com/w3c/browser-specs/issues/1383 for the spec issue. Usually specs get added within a few days. If you don't want to wait, you will need to add to the exception list.

File: http/headers/Priority.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
+{
+  "http": {
+    "headers": {
+      "Priority": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/HTTP/Headers/Priority",
```
Comment (Elchi3) [inline http/headers/Priority.json:6]: Content page does not yet exist and we don't want to generate 404 links for BCD's consumers. I will be added automatically once the docs have been published.

```suggestion
```

**Comment by hamishwillee** [comment]
Thanks @Elchi3 - clearly disappearing for a few days on other work was beneficial :-)

**Pull request merged**